.env.production

```diff
@@ -1,8 +1 @@
-BASE_URL=null
-LMS_BASE_URL=null
-LOGIN_URL=null
-LOGOUT_URL=null
-CSRF_TOKEN_API_PATH=null
-REFRESH_ACCESS_TOKEN_ENDPOINT=null
-ACCESS_TOKEN_COOKIE_NAME=null
-USER_INFO_COOKIE_NAME=null
```

Comment:
I am not sure if we need to remove this? since these variables would still be coming from internals but only whitelisted by Gatsby.

Reply:
Probably not really necessary to keep these around (safe to remove), but might be good to be explicit about the production variables we expect. Let's keep these variables in `.env.production` for now.

Reply:
It looks like any variable names added in the whitelist are looked up as `process.env.X` where X is the variable name, so my understanding is that these would be that these would not be required. I suppose we'd find out quickly when this hits stage if they are required though ;)